pkg/asset/machines/master.go

```diff
@@ -253,12 +311,28 @@ func (m *Master) Files() []*asset.File {
 		files = append(files, m.UserDataFile)
 	}
 	files = append(files, m.MachineConfigFiles...)
+	// Add the host CRD so we don't block trying to create a type
+	// unknown to the cluster.
+	files = append(files, m.CRDFiles...)
```

Comment:
this doesn't seem to be defined... _confused_

Reply:
I'm not sure what isn't defined? m.CRDFiles is defined as part of the Master type.

Reply:
this change is across 2 commits...

Reply:
Ah, I must have messed something up when I rebased. I'll fix that.